.github/workflows/publish-engine.yml

```diff
@@ -0,0 +1,28 @@
+name: publish-engine
+on:
+  push:
+    branches:
+      - main
```

Comment:
This will trigger the workflow on every merge to main.

Do we want that?

Reply:
It seemed like we may as well since it will allow us to do local testing of SDKs against pushed-but-not-released engine builds. It's not strictly necessary though, just seemed nice-to-have.

I guess the downside is we will be pushing a lot more data to our ghcr repo. If that's a significant downside or there's other ones I'll get rid of this, let me know.

Reply:
👍